daemon/daemon.go

```diff
@@ -1163,7 +1163,6 @@ func NewDaemon(dp datapath.Datapath) (*Daemon, *endpointRestoreState, error) {
 			return nil, nil, fmt.Errorf("unable to setup workload: %s", err)
 		}
 
-		log.Infof("Container runtime options set: %s", workloads.GetRuntimeOptions())
```

Comment:
I'd suggest to remove the other log statement, as the settings are relevant if workloads are enabled.